packages/ag-charts-enterprise/src/series/sankey/sankeySeries.ts

```diff
@@ -524,4 +524,22 @@ export class SankeySeries extends FlowProportionSeries<
     override getLabelData(): _Util.PointLabelDatum[] {
         return [];
     }
+
```

Comment:
I think that these two `computeFocusBounds` methods are fine, but apparently SonarCloud is upset about code duplication. Consider adding abstract like `computeNodeFocusBounds` and `computeLinkFocusBounds` that the derived classes can override.

Reply:
All shapes already have a definite bounding box. But the keyboard navigation operates on datums not shapes. There could perhaps be a method like `getShapeOfDatum(datumIndex): Shape` that retrieves the shape of datum.